somewhy php devs have embedded magic database into php extension. yuck!

rebuild data file for now
--- php-5.3.20/ext/fileinfo/Makefile.frag	2012-12-19 08:55:19.000000000 +0200
+++ php-5.3.20/ext/fileinfo/Makefile.frag	2013-01-16 11:11:51.444547648 +0200
@@ -1,2 +1,5 @@
 
 $(builddir)/libmagic/apprentice.lo: $(srcdir)/data_file.c
+MIME_MAGIC_DB ?= /usr/share/misc/magic.mgc
+$(srcdir)/data_file.c: $(MIME_MAGIC_DB) $(SAPI_CLI_PATH)
+	$(SAPI_CLI_PATH) $(srcdir)/create_data_file.php $< > $@
